README.md

```diff
@@ -37,39 +42,42 @@ server.whoami((err, feed) => {
 
 ### `require('ssb-config')`
 
-returns you the stock standard config for starting an `ssb-server`
+Returns you the stock stanard config for starting an __ssb-server__.
 
 ### `require('ssb-config/inject')(appName, opts) => Object`
 
 A function which takes:
-- `appName` *(string)* which declares where to look for further config, where to read and write databases. Stores data in `~/.${appName}`, defaults to `ssb` (so data in `~/.ssb`).
-- `opts` *(object)* an object which can override config defaults (see 'Configuration' below)
+- `appName` *(string)* Which declares where to look for further config, where to read and write databases. Stores data in `~/.${appName}`, defaults to `ssb` (so data in `~/.ssb`).
+- `opts` *(object)* An object which can override config defaults (see *[Configuration](#configuration)* below).
 
 ## Configuration
 
-All configuration is loaded via [`rc`](https://github.com/dominictarr/rc). This means the final config is a result of config collected from `opts` passed into the inject method, cli args, env var, and config (e.g. `~/.ssb/config`). See the rc repo for full details.
+All configuration is loaded via `rc`. This means the final config is a result of config collected from `opts` passed into the inject method, cli args, env var, and config (e.g. `~/.ssb/config`). See the [rc repo](https://github.com/dominictarr/rc) for full details.
 
-Options:
-* `connections` *(object)* Details `incoming` and `outgoing` connections behaviour (see below)
-* `remote` ... TODO ... a multisever address for ... (in the future this may be deprecated / derived from `connections`
+__Options__
+* `connections` *(object)* Details `incoming` and `outgoing` connections behaviour ([See below](#connections)).
+* `remote` *(string)* Multiserver address to connect as a client. Useful in some cases, such as using [ssb-unix-socket](https://github.com/ssbc/ssb-unix-socket) + [ssb-no-auth](https://github.com/ssbc/ssb-no-auth). In the future this may be deprecated / derived from `connections`.
```

Comment:
Not sure if this is the right place to mention this (but it has bitten me before), when entering the public key in the multiserver address, you need to make sure you don't have the public key type in there, as otherwise ssb-client will fail with cryptic errors. Most probably a bug, but until it gets fixed, might be good to mention here.

So, the multiserver address has to be something like `ws://127.0.0.1:8989~shs:@FlzOOyrhTG+Cl6YND5tReJtd2ufUj3++F0ZsfcPalRc=` (missing `.ed25519` that seems to be optional in all other places but in a multiserver address)

Reply:
Good catch @victorb, I added the link to the definition of multiserver address.
The address you were trying (with the key type at the end) is not considered a valid address.